### ISSUE TITLE
Build with "noImplicitAny": true fails

### ISSUE THREAD

**Issue by swftvsn** [firebase/firebase-functions#213]
### Version info
Typescript 2.8.1

**firebase-functions:**
1.0.0

**firebase-tools:**
3.18.0

**firebase-admin:**
5.11.0

### Steps to reproduce
Use above versions and try to build. Used tsconfig.json:

```json
{
  "compilerOptions": {
    "lib": ["es6", "es2015.promise", "dom"],
    "module": "commonjs",
    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": false,
    "outDir": "build",
    "sourceMap": false,
    "target": "es6",
    "typeRoots" : ["./src/_shared-functions/types", "./src/_shared/types", "./node_modules/@types"]
  },
  "include": [
    "src/**/*.ts",
    "spec/**/*.ts"
  ]
}
```

### Were you able to successfully deploy your functions?
Can't build.

### Expected behavior
I can build.

### Actual behavior

Running `node_modules/typescript/bin/tsc` produces error

```bash
node_modules/firebase-functions/lib/cloud-functions.d.ts(62,58): error TS7006: Parameter 'any' implicitly has an 'any' type. 
```
The offending file contains:

```ts
export declare namespace Change {
    /** Factory method for creating a Change from a `before` object and an `after` object. */
    function fromObjects<T>(before: T, after: T): Change<T>;
    /** Factory method for creating a Change from a JSON and an optional customizer function to be
     * applied to both the `before` and the `after` fields.
     */
    function fromJSON<T>(json: ChangeJson, customizer?: (any) => T): Change<T>;
}
```

Which translates to this line: 
https://github.com/firebase/firebase-functions/blob/0a0aea9fca234b12e182f4971fc6e4621bc4bff2/src/cloud-functions.ts#L120

Which should probably read 
```ts
export function fromJSON<T>(json: ChangeJson, customizer: (a: any) => T = reinterpretCast): Change<T> {
```

Or something similar?

I can build with `"noImplicitAny": false` fine though, so there's no rush.

**Comment by wcandillon** [comment]
@laurenzlong When will this update be available?

**Comment by laurenzlong** [comment]
It was just made available in 1.0.2

**Comment by wcandillon** [comment]
@laurenzlong Yes that works, thank you, really appreciate it 🙌🏻
